service/utils.go

```diff
@@ -217,5 +217,13 @@ func checkUserPermissions(w http.ResponseWriter, r *http.Request) (string, error
 	}
 
 	// Get the user from the token
-	return token.Claims[usso.ClaimsUsername].(string), nil
+	username := token.Claims[usso.ClaimsUsername].(string)
+
+	// Check that the role is at least an Admin
+	role := datastore.Environ.DB.RoleForUser(username)
+	if role < datastore.Admin {
```

Comment:
retricting to Admin or more is another reason to modify method name. Though now I'm not sure what name is suitable, because this does some different things...
YoGimmeDaUserCrapBro()
AllowUser() 
checkUserPermissions()

ok, accepted checkUserPermissions, BUT...
modify comments of the method, explaining better than the method validates Admin and Superusers, and explaining the return value and what implies returning an empty username